cluster/addons/addon-manager/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### Version 8.1  (Sun October 22 2017 zou nengren @zouyee)
```

Comment:
Can you update the date here and bump the version from 1.8.1 --> 1.8.4?

Reply:
got it!